test/verify/check-services

```diff
@@ -188,6 +189,270 @@ WantedBy=default.target
+        b.wait_visible("#create-timer")
+        b.click('#create-timer')
+        b.wait_popup("timer-dialog")
+        b.set_val("#servicename", "hourly_timer")
+        b.set_val("#description", "Hourly timer")
+        b.set_val("#command", "/bin/sh -c 'while true; do sleep 5; done'")
+        b.click("#boot-or-specific-time button")
+        b.click("#boot-or-specific-time .dropdown-menu li[value=2] a")
+        b.click("#drop-repeat button")
+        b.click("#drop-repeat .dropdown-menu li[value=60] a")
+        b.click("[data-content='add'][data-index=0]")
+        b.set_val("[data-index=0][data-content='minutes']", "05")
+        b.set_val("[data-index=1][data-content='minutes']", "26")
+        b.click("#timer-save-button")
+        b.wait_popdown("timer-dialog")
+        b.wait_present(svc_sel('hourly_timer.timer'))
+
+        m.execute("timedatectl set-time '2020-04-10 03:00:00'")
+        m.execute("systemctl start hourly_timer.timer")
+        b.wait_present(svc_sel('hourly_timer.timer'))
+        b.wait_in_text(svc_sel('hourly_timer.timer'), "Hourly timer")
+        self.assertIn("Fri 2020-04-10 03:05", m.execute("LC_ALL=C systemctl list-timers"))
+        m.execute("timedatectl set-time '2020-04-10 03:07:00'")
+        b.wait_present(svc_sel('hourly_timer.timer'))
+        self.assertIn("Fri 2020-04-10 03:26", m.execute("LC_ALL=C systemctl list-timers"))
+        m.execute("timedatectl set-time '2020-04-10 04:00:00'")
+        b.wait_present(svc_sel('hourly_timer.timer'))
+        # checks if timer runs on next hour at 5 min and 26 min
+        self.assertIn("Fri 2020-04-10 04:05", m.execute("LC_ALL=C systemctl list-timers"))
+        m.execute("timedatectl set-time '2020-04-10 04:10:00'")
+        b.wait_present(svc_sel('hourly_timer.timer'))
+        self.assertIn("Fri 2020-04-10 04:26", m.execute("LC_ALL=C systemctl list-timers"))
+        m.execute("systemctl stop hourly_timer.timer")
+        # creates a new timer that runs at today at 23:59
+        b.wait_visible("#create-timer")
+        b.click('#create-timer')
+        b.wait_popup("timer-dialog")
+        b.set_val("#servicename", "no_repeat_timer")
+        b.set_val("#description", "No repeat timer")
+        b.set_val("#command", "/bin/sh -c 'while true; do sleep 5; done'")
+        b.click("#boot-or-specific-time button")
+        b.click("#boot-or-specific-time .dropdown-menu li[value=2] a")
+        b.set_val("#hr", "23")
+        b.set_val("#min", "59")
+        b.click("#timer-save-button")
+        b.wait_popdown("timer-dialog")
+        b.wait_present(svc_sel('no_repeat_timer.timer'))
+        b.wait_in_text(svc_sel('no_repeat_timer.timer'), "No repeat timer")
+
+        m.execute("timedatectl set-time '2020-04-10 04:10:00'")
+        m.execute("systemctl start no_repeat_timer.timer")
+        b.wait_present(svc_sel('no_repeat_timer.timer'))
+        self.assertIn("Fri 2020-04-10 23:59", m.execute("LC_ALL=C systemctl list-timers"))
```

Comment:
see above, probably a race condition (maybe it would be good to write a function to cover waiting for this info)